raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/RawSpansGroupingTransformer.java

```diff
@@ -71,6 +83,17 @@ public void init(ProcessorContext context) {
   public KeyValue<String, StructuredTrace> transform(TraceIdentity key, RawSpan value) {
     ValueAndTimestamp<RawSpans> rawSpans = inflightTraceStore.get(key);
     RawSpans agg = rawSpans != null ? rawSpans.value() : RawSpans.newBuilder().build();
+    if (maxSpanCountMap.containsKey(key.getTenantId()) && agg.getRawSpans().size() > maxSpanCountMap.get(key.getTenantId())) {
+      if (logger.isDebugEnabled()) {
+        logger.debug("Number of spans in tenant_id={}, trace_id={} is {}",
```

Comment:
Not sure this ling and the one in line 92 should be separated logs as one without the each isn't actionable.

Reply:
merged both messages into one.